src/vs/workbench/services/textfile/common/textFileService.ts

```diff
@@ -774,8 +796,7 @@ export class TextFileService extends Disposable implements ITextFileService {
 	}
 
 	private suggestFileName(untitledResource: URI): URI {
-		const untitledFileName = this.untitledEditorService.suggestFileName(untitledResource);
-
+		const untitledFileName = this.untitledEditorService.suggestFileName(untitledResource) || 'untitled';
```

Comment:
@mjbvz not a big fan, I suggest we make `suggestFileName` never return `null` by providing a good value in case the model does not exist.

Reply:
Done